less/index.less

```diff
@@ -53,7 +53,220 @@ table {
     box-sizing: border-box;
 }
 
+
+// Variables
+@standardpadding: 5%;
+@paddingsmall: 1%;
+@mobilebreakpoint: 500px;
+@themecolor: #A96A67;
+
+// Mixins
+.displayFlex(@justify, @align, @direction){
+	display: flex;
+	justify-content: @justify;
+	align-items: @align;
+	flex-direction: @direction;
+}
+
+.testborder {
+	border: 1px solid blue;
+}
+
+.themeborder {
+	border: dashed 2px @themecolor;
+}
+
+
+
 html, body {
-    height: 100%;
+	height: 100%;
+	font-size: 62.5%;
+	font-family: 'Nanum Gothic', sans-serif;
+}
+
+h1, h2, h3, h4, h5, h6 {
+	font-weight: 530;
+	padding-bottom: @paddingsmall;
+}
+
+h1 {
+	font-size: 5rem;
+}
+
+h2{
+	font-size: 2rem;
+}
+
+h3{
+	font-size: 1.5rem;
+}
+
+ul {
+	list-style: square inside none;
+	li {
+	padding-bottom: 3px;
+	}
+}
+
+.container {
+	max-width: 880px;
+	font-size: 1.25rem; 
+	margin: 0 auto;
+	border: dashed 2px @themecolor;
+	border-bottom: 0;
+
+	@media (max-width: @mobilebreakpoint){
```

Comment:
a common way to do this is to define the @mobilebreakpoint : ~"(max-width : 500px)"
if you do it this way, then you don't have to write it out explicitly.